Jenkinsfile

```diff
@@ -0,0 +1,29 @@
+#!groovy
+
+// https://github.com/feedhenry/fh-pipeline-library
+@Library('fh-pipeline-library') _
+
+fhBuildNode() {
+    stage('Install Dependencies') {
+        npmInstall {}
+    }
+
+    stage('Lint') {
+        sh 'grunt eslint'
+    }
+
+    stage('Unit Tests') {
+        withOpenshiftServices(['mongodb']) {
+            sh "node ./test/setup_mongo_wendy.js"
+            sh 'grunt fh:test'
+            input ""
```

Comment:
Remove the input

Reply:
@AdamSaleh This is stuck here at the moment https://jenkins-wendy.ci.feedhenry.org/job/feedhenry/job/fh-forms/job/PR-24/13/